acoustid.py

```diff
@@ -20,13 +20,16 @@
 import requests
 import contextlib
 import errno
+
```

Comment:
If you don't mind too terribly much, would you mind reverting these whitespace changes? They'll make the diff a little smaller.

Reply:
I knew how, I'd love too

Reply:
It's OK to just delete the lines in a new commit!